Sign_Up_Demo/View/CollectionView/FooterCollectionReuseableView.swift

```diff
@@ -0,0 +1,9 @@
+import UIKit
+
+class FooterCollectionReuseableView: UICollectionReusableView {
+
+    // MARK: - LifeCycle Methods
+    override func awakeFromNib() {
```

Comment:
Remove if usless.

Reply:
Class is being used and but method inside it is not required so I removed it.